### ISSUE TITLE
cannot connect to X server & output video broken

### ISSUE THREAD

**Issue by brianw0924** [hysts/pytorch_mpiigaze_demo#10]
Hi, I'm trying to run this demo on Colab

I run the command:

!ptgaze --mode mpiigaze --video video.mp4 --o ./pytorch_mpiigaze_demo/assets/results/

and get the error. The video still produced, but it's broken (all black, and always 00:00)

and I want to ask will this demo work if my input video is crop to only eyes? 

INFO:ptgaze.main:mode: MPIIGaze
device: cpu
model:
  name: resnet_preact
face_detector:
  mode: mediapipe
  dlib_model_path: /root/.ptgaze/dlib/shape_predictor_68_face_landmarks.dat
  mediapipe_max_num_faces: 3
gaze_estimator:
  checkpoint: /root/.ptgaze/models/mpiigaze_resnet_preact.pth
  camera_params: /tmp/camera_params.yaml
  use_dummy_camera_params: true
  normalized_camera_params: /usr/local/lib/python3.7/dist-packages/ptgaze/data/normalized_camera_params/mpiigaze.yaml
  normalized_camera_distance: 0.6
demo:
  use_camera: false
  display_on_screen: true
  wait_time: 1
  image_path: null
  video_path: video.mp4
  output_dir: pytorch_mpiigaze_demo/assets/results
  output_file_extension: avi
  head_pose_axis_length: 0.05
  gaze_visualization_length: 0.05
  show_bbox: true
  show_head_pose: false
  show_landmarks: false
  show_normalized_image: false
  show_template_model: false
PACKAGE_ROOT: /usr/local/lib/python3.7/dist-packages/ptgaze

INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
: cannot connect to X server

**Comment by hysts** [comment]
@brianw0924 

Ah, ok, so that's why you chose to use the MPIIGaze model. If the model doesn't work for your input, you might want to check if the way the eye region is cropped is the same as the way the model was trained.
Also, due to the limited variation of the training data, the MPIIGaze model can only handle a limited range of face orientations and gaze directions, so that may be the cause of the problem.

**Comment by hysts** [comment]
@brianw0924 

Yes, I think that's a good starting point.
